values-es/strings.xml

```diff
@@ -1,83 +1,83 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name">Wavelet</string>
-    <string name="no_active_sessions">No hay sesiones abiertas todavía</string>
-    <string name="single_user_warning">Favor de instalar esta aplicación como usuario de sistema</string>
+    <string name="no_active_sessions">No hay aplicaciones abiertas todavía</string>

```

Comment:
I'm afraid the meaning already got lost in translation before. It should be 'No music sessions have been started yet'.